### ISSUE TITLE
Metamodel - binaire relaties

### ISSUE THREAD

**Issue by mickbaggen** [Geonovum/MIM-Werkomgeving#159]
In vergelijking met het metamodel in NEN 2660-1 (Ontw.) en het UML metamodel lijkt het MIM metamodel zich te beperken tot het gebruik van binaire relatiesoorten (View 3b, ). Waarom is deze keuze gemaakt?

En waarom is in View 3b (diagram Relatierol) de "Associationclass" (gestippelde lijn) op de relatie tussen Relatiesoort en Objecttype getrokken naar Relatierol, en niet naar "Relatierol doel" ?

**Comment by lennartvanbergen-kadaster** [comment]
Binaire relaties? Van A naar B of C en niet van A naar tegelijk B en C?

**Comment by mickbaggen** [comment]
Inderdaad, binaire relaties hebben altijd twee uiteinden (relatierollen). In RDF (SKOS/RDFS/OWL/SHACL) komen enkel binaire relaties voor, in UML, NEN 2660-1 en (bijvoorbeeld) NIAM/ORM/FCO-IM kunnen ook n-aire relaties voorkomen (n >2), dus meer dan 2 rollen in een relatie.

**Comment by lennartvanbergen-kadaster** [comment]
MIM zegt niet dat alleen binaire relaties zijn toegestaan. Het is wel zo dat alleen binaire zijn beschreven. 

Het officieuze criterium voor opname is dat meer dan 1 organisatie het gebruikt, zolang maar 1 het gebruikt kan die organisatie dat in zijn eigen extensie opnemen en inbrengen (vooralsnog is deze niet ingebracht).  

In UML en andere modelleertalen bestaan idd wel tertiaire relaties, of n-aire maar vrijwel niemand gebruikt die ooit dus dat hebben we vooralsnog niet opgenomen.

**Comment by JanCampschroer** [comment]
ik wil deze wel doen

**Comment by JanCampschroer** [comment]
We zouden deze toelichting op kunnen nemen bij 2.3.3 Relatieklasse. Dat je die kunt gebruiken om tertiaire relaties vast te leggen. Maar dat je ook eigen aanvulling kunt opnemen. Of ze gelijk 'objectiveren'. Is allemaal niet 100% maar dan kun je wel voor uit.

**Comment by lennartvanbergen-kadaster** [comment]
Dit is een y-issue, geen z. Voorstel om dit mee te nemen met een grotere ronde.
